packages/chrome.extensions.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,39 @@
+$ErrorActionPreference = 'Stop'
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+
+  # Important: Chrome should be opened with internet access for
```

Comment:
the extensions should be loaded during the package installation, is it enough with launching Chrome after the registry modification? is there something created during loading that we can check afterwards to ensure they are loaded correctly?

Reply:
Launching it for a minute is enough, Chrome downloads the extensions to `%localappdata%\Google\Chrome\User Data\Default\Extensions`.

Reply:
Good! can you please add to this script starting chrome and waiting for the extensions to be in that directory?

Reply:
Done.